longevity_test.py

```diff
@@ -576,10 +576,14 @@ def get_email_data(self):
         except Exception as error:  # pylint: disable=broad-except
             self.log.error("Error in gathering Grafana screenshots and snapshots. Error:\n%s", error)
 
+        benchmarks_results = self.db_cluster.get_node_benchmarks_results() if self.db_cluster else {}
+        # If cluster was not created, not need to collect nemesis stats - they do not exist
+        nemesises_stats = self.get_nemesises_stats() if self.db_cluster else ''
```

Comment:
`get_nemesises_stats()` returns a dict.

and, btw, plural form of *nemesis* is *nemeses* :)

Reply:
fixed